ChatRPG/ChatRPG/Pages/Campaign.razor

```diff
@@ -85,10 +89,9 @@
         return response.Choices[0].Message.Content;
     }
 
-    async Task ScrollToElement(string elementId)
+    private async Task ScrollToElement(string elementId)
     {
-        IJSObjectReference module = await JsRuntime.InvokeAsync<IJSObjectReference>("import", "./js/scroll.js");
-        await module.InvokeVoidAsync("ScrollToId", elementId);
+        await _scrollJsScript!.InvokeVoidAsync("ScrollToId", elementId);
```

Comment:
maybe null coalesce for god ordens skyld, for at undgå et crash hvis scriptet ikke er loadet når den skal autoscrolle (unlikely, but still)
```suggestion
        await _scrollJsScript?.InvokeVoidAsync("ScrollToId", elementId);
```

Reply:
Man kan ikke bare tilføje `?` i stedet for `!`. Hvis den skal kunne coalesce skal give den en fallback value og det kan vi ikke i det her tilfælde. 
En fallback value ville se nogenlunde sådan her ud:
`await (_scrollJsScript ?? VALUE).InvokeVoidAsync("ScrollToId", elementId);`
men vi har ikke noget at give den i dette tilfælde. I øvrigt vil den aldrig være null da vi loader scriptet på page init og funktionen `ScrollToELement` vil aldrig blive kaldt uden at siden er loaded. 

Hvis vi skal sikre det helt, så kan vi tjekke om `_scrollJsScript` er null og hvis den er, så kan vi køre `OnInitializedAsync`, men det føles virkelig dirty.

Reply:
Alright, ellers kan vi vel lave den klassiske:
```csharp
if (_scrollJsScript is not null)
{
    ...
}
```
Men god pointe med at `OnInitializedAsync` altid vil være kaldt først. Jeg resolver denne kommentar uanset.